charts/kueue/README.md

```diff
@@ -24,12 +24,21 @@ Quick start instructions for the setup and configuration of kueue using Helm.
 
 ##### Install chart using Helm v3.0+
 
+Either clone the kueue repository:
+
 ```bash
 $ git clone git@github.com:kubernetes-sigs/kueue.git
 $ cd kueue/charts
 $ helm install kueue kueue/ --create-namespace --namespace kueue-system
 ```
 
+Or use the charts pushed to `us-central1-docker.pkg.dev/k8s-staging-images/charts/kueue`:
+
+```bash
+VERSION=v0.9.0
```

Comment:
Could you please add file path for this file here

https://github.com/kubernetes-sigs/kueue/blob/43b7374764498604e447c312b36bae521c472f42/Makefile#L282

to replace version on `make prepare-release-branch`.

Reply:
How about https://github.com/kubernetes-sigs/kueue/pull/3517/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R283?

Reply:
Tested with `$ sed -r 's/--version="v[0-9]+\.[0-9]+\.[0-9]+/--version="v0.9.1/g' -i charts/kueue/README.md`

```
$ gd
diff --git a/Makefile b/Makefile
index dcf8edbb..0e343dd8 100644
--- a/Makefile
+++ b/Makefile
@@ -280,6 +280,7 @@ artifacts: kustomize yq helm ## Generate release artifacts.
 .PHONY: prepare-release-branch
 prepare-release-branch: yq kustomize ## Prepare the release branch with the release version.
        $(SED) -r 's/v[0-9]+\.[0-9]+\.[0-9]+/$(RELEASE_VERSION)/g' -i README.md -i site/hugo.toml -i charts/kueue/README.md
+       $(SED) -r 's/--version="v[0-9]+\.[0-9]+\.[0-9]+/--version="$(RELEASE_VERSION)/g' -i charts/kueue/README.md
        $(YQ) e '.appVersion = "$(RELEASE_VERSION)"' -i charts/kueue/Chart.yaml
        @$(call clean-manifests)

diff --git a/charts/kueue/README.md b/charts/kueue/README.md
index fbcb94d9..6609cec9 100644
--- a/charts/kueue/README.md
+++ b/charts/kueue/README.md
@@ -35,7 +35,7 @@ $ helm install kueue kueue/ --create-namespace --namespace kueue-system
 Or use the charts pushed to `us-central1-docker.pkg.dev/k8s-staging-images/charts/kueue`:

 ```bash
-helm install kueue oci://us-central1-docker.pkg.dev/k8s-staging-images/charts/kueue --version="v0.9.0" --create-namespace --namespace=kueue-system
+helm install kueue oci://us-central1-docker.pkg.dev/k8s-staging-images/charts/kueue --version="v0.9.1" --create-namespace --namespace=kueue-system
 ```
```

Reply:
Thanks!